### DISCUSSION TITLE
[BUG] Synced but solving old challanges

### DISCUSSION THREAD

**Discussion by Romijoxx** [Chia-Network/chia-blockchain#5004] [Support]
**Describe the bug**
My Fulll Node is running and synced (as you can see in the Screen) I am connected to a lot of other full nodes, too.  (Screenshot is 5 minutes old)
![grafik](https://user-images.githubusercontent.com/17925930/118283976-3af0f800-b4d0-11eb-8a00-b5a5062d75cd.png)

Nevertheless sind twenty hourse it trys to solve random challanges. All around from 4h to 20 min olds but no new ones, 

**To Reproduce**

Unknown

**Expected behavior**
Solving new Challanges that are a few seconds old

**Desktop (please complete the following information):**

- OS: Win 10 Pro
- CPU: I9-9900k

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
